daemon/src/wire.rs

```diff
@@ -25,8 +25,8 @@ pub enum TakerToMaker {
 #[allow(clippy::large_enum_variant)]
 pub enum MakerToTaker {
     CurrentOrder(Option<Order>),
-    // TODO: Needs RejectOrder as well
     ConfirmTakeOrder(OrderId), // TODO: Include payout curve in "accept" message from maker
+    RejectTakeOrder(OrderId),
```

Comment:
Along with the other naming above, I think dropping the `Take` here would be better.

The taker is placing an order yes? Then saying `ConfirmOrder` and `RejectOrder` should be correct?